exonum/benches/criterion/transactions.rs

```diff
@@ -12,33 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const TRANSACTIONS_COUNT: usize = 40_000;
-const SAMPLE_SIZE: usize = 3;
+const TRANSACTIONS_COUNT: usize = 1_000;
```

Comment:
why we need this significant decrease of the transaction count?

Reply:
This is my change ✋ Unless there are reasonable arguments for huge transaction count (i.e., unless we _expect_ the timing growth with the transaction could to be super-linear or sub-linear), I see no particular reason to use it. The transaction count allowing for reasonable number of samples, and correspondingly higher accuracy could be more important. More specifically, I'd be against decreasing the sample size to `< 10`, which corresponds to `TRANSACTIONS_COUNT ~ 5_000` for the reasonable running time (~30s / bench on my laptop).

Reply:
Ok, thanks, can we add some docs about this constant, that nobody will ask a question in future?

Reply:
In  my opinion large amount of transaction helps to decrease influence of implicit  overheads.

Reply:
@alekseysidorov Would you be more satisfied with the following settings?

```rust
const TRANSACTIONS_COUNT: usize = 4_000;
const SAMPLE_SIZE: usize = 10;
```

I'm not sure decreasing sample size further will do us good; the decrease from 20 to 10 increases st.dev. of results ~2 times if I understand the measuring methodology correctly.

To be frank, I don't think the benchmark in the present form has significant non-linear overheads (it *may* have had them when it included writing transactions to the DB), but I didn't investigate this question.